clients/cloud/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+group = "io.confluent.ccloud"
+version = "5.0.1"
```

Comment:
What does this `version` refer to?

Reply:
I copied this from other examples, namely java

Reply:
ie, will it change with each version of CP?  I don't think so...

Reply:
cc: @confluentinc/tools 
they get auto-updated in many maven files..

I initially questioned the use of gradle, but it seems preferred by kotlin people.

Reply:
will use default (0.0.1-SNAPSHOT)

Reply:
@mhowlett i went full blown purist here - gradle build using Kotlin DSL to build kotlin app :)